assets/js/admin.js

```diff
@@ -50,6 +60,39 @@
             $('.tab-' + which).addClass('active');
         });
 
+        // Loads more logs when 'Load More' button is clicked.
+        $(document).on('click', '.load-more', function() {
+
+          var data = {
+            action: 'postmark_load_more_logs',
+            offset: logs_offset,
+            _wpnonce : $('#_wpnonce').val()
+          };
+
+          // Prepares for next 'Load More' by increasing the offset amount for next query.
+          logs_offset += 10;
+
+          $.post( ajaxurl, data, function( response ) {
+
+            // Parses response string into JSON.
+            response = JSON.parse(response);
+
+            // Adds new logs to logs table.
+            $('#pm-log-table tr:last').after(response.html);
+
+            // Tracks the number of total logs present in logs table.
+            var total_logs_count = response.total_count;
```

Comment:
Getting totals from large tables can result in locking and other issues. Instead of getting "total_count", I would return a "has_more" boolean that is true if the there were at least the number of items requested in the last batch (i.e. if the search turns up < 10 when you request 10, then you return "has_more == false", otherwise, true. And then the UI is simply "load more" until there are none left.

Reply:
Switched to the has_more method. Nice one!